packages/version_repository_interface/lib/src/utils/version_string.dart

```diff
@@ -0,0 +1,24 @@
+/// Extension for [String] to check if it is a semantic version.
+extension VersionString on String {
+  /// Check if the string is a semantic version.
+  bool isSemanticVersion() {
```

Comment:
Could we have 1 simple test for this method in the tests folder

Reply:
done :)